eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -12,6 +12,7 @@ param (
   [String]$ServiceDirectory,
   [Parameter(Mandatory = $true)]
   [String]$PackageName,
+  [String]$ChangelogPath,
```

Comment:
I assume you also need to update the calling functions somewhere to pass the changelog path.

Reply:
I remove this change for now